index.html

```diff
@@ -1498,70 +1498,64 @@ <h2>
 
 <span class="comment">...</span>
 }
-</pre>
+      </pre>
+
       <p>
-The algorithm to use when processing a <code>publicKey</code>
-property in a <a>DID document</a> is:
+The steps to use when processing a <code>publicKey</code> property in a
+<a>DID document</a> are:
       </p>
 
       <ol class="algorithm">
         <li>
-Let <em>value</em> be the data associated with the
-        <code>publicKey</code> property and initialize <em>result</em> to
-        <code>null</code>.
+Let <em>value</em> be the data associated with the <code>publicKey</code>
+property and initialize <em>result</em> to <code>null</code>.
         </li>
 
         <li>
 If <em>value</em> is an object, the key material is embedded. Set
-        <em>result</em> to <em>value</em>.
+<em>result</em> to <em>value</em>.
         </li>
 
         <li>
-If <em>value</em> is a string, the key is included by reference.
-Assume <em>value</em> is a URL.
+If <em>value</em> is a string, the key is included by reference. Assume
+<em>value</em> is a URL.
           <ol>
             <li>
-Dereference the URL and retrieve the <code>publicKey</code>
-properties associated with the URL (e.g., process the
-<code>publicKey</code> property at the top-level of the
-dereferenced document).
+Dereference the URL and retrieve the <code>publicKey</code> properties
+associated with the URL. For example, process the <code>publicKey</code>
+property at the top-level of the dereferenced document.
             </li>
 
             <li>
-Iterate through each public key object.
-              <ol>
-                <li>
-If the <code>id</code> property of the object matches
-<em>value</em>, set <em>result</em> to the object.
-                </li>
-              </ol>
+Iterating through each public key object, if the <code>id</code> property of the
+object matches <em>value</em>, set <em>result</em> to the object.
             </li>
           </ol>
         </li>
 
         <li>
 If <em>result</em> does not contain at least the <code>id</code>,
-<code>type</code>, and <code>controller</code> properties as well as any
-mandatory public cryptographic material, as determined by the
-<em>result</em>'s <code>type</code> property, throw an error.
+<code>type</code>, and <code>controller</code> properties, as well as any
+mandatory public cryptographic material, as determined by the <code>type</code>
+property of <em>result</em>, throw an error.
```

Comment:
```suggestion
property of the <em>result</em>, throw an error.
```

Reply:
I don't agree with this change. All the preceding steps refer to _result_ as a parameter (e.g., "Set _result_ to _value_.", and so on).

Reply:
Ah, OK.